java-package/brainflow/src/main/java/brainflow/BrainFlowInputParams.java

```diff
@@ -16,6 +16,8 @@
     public int timeout;
     public String serial_number;
     public String file;
+    public int metric;
+    public int classifier;
```

Comment:
it should not be here

Reply:
Input params needs to save metric and classifieier so we can have it as a parameter.